build/Linux+BSD/portable/copy-libs

```diff
@@ -79,8 +79,9 @@ getCrossPlatformDependencies() {
   copyLib libQt5Sql.so.5
   copyLib libQt5Svg.so.5
   #copyLib libQt5WebChannel.so.5  # Not present on Debian, and apparently not needed
-  copyLib libQt5WebKit.so.5
-  copyLib libQt5WebKitWidgets.so.5
+  copyLib libQt5WebEngine.so.5
+  copyLib libQt5WebEngineCore.so.5
+  copyLib libQt5WebEngineWidgets.so.5
```

Comment:
They would get reported if they are linked to by one of the Qt libraries that is already included in the AppImage. For example:
- mscore depends on libA.so for feature X
- libA.so depends on libB.so for feature Y (but not for feature X)

In this case libB.so will be reported as missing by check-depends even though it is not actually needed by MuseScore. Probably one of the general Qt libs (e.g. libQt5Core.so) links to lots of other Qt libs.

Usually indirect dependencies will be needed, hence why they are reported as missing by check-depends. I'm planning on making an ignore list to avoid reporting ones we know we don't need.